app/views/distributions/leap-15.0.html.erb

```diff
@@ -93,6 +93,19 @@
               <a href="https://download.opensuse.org/distribution/leap/15.0/live/openSUSE-Leap-15.0-KDE-Live-x86_64-Current.iso.sha256" class="card-link"><%= _('Checksum') %></a>
             </div>
           </div><!-- /.card -->
+          <div class="card">
+            <div class="card-body">
+              <h4 class="card-title">
+                <span class="typcn typcn-download-outline"></span>
+                Rescue LiveCD (650MB)
+              </h4>
+              <h6 class="card-subtitle mb-2 text-muted"><%= _("For CD and USB stick") %></h6>
+              <a href="https://download.opensuse.org/distribution/leap/15.0/live/openSUSE-Leap-15.0-Rescue-CD-x86_64-Current.iso" class="card-link"><%= _('Direct Link') %></a>
```

Comment:
this route (`https://download.opensuse.org/distribution/leap/15.0/live/openSUSE-Leap-15.0-Rescue-CD-x86_64-Current.iso`) is repeated 4 times, could it make sense to assign it to a variable, so that it is only needed to modify it once in case that it changes?